core/datasource/external/ethcall/engine.go

```diff
@@ -36,6 +36,10 @@ import (
 	"github.com/ethereum/go-ethereum"
 )
 
+// if we haven't sent a contract call event in this many e.blockIntervals we'll send a dummy one so that
+// core knows of the latest eth block height we've checked.
+var heatbeatInterval = time.Hour
```

Comment:
Should we have some sort of configuration in the config file which a long weird explicit name (like some others) so we can set it easily to something small so we can see it in like the system tests quickly, but default it to 1hr for mainnet?

Reply:
Can't hurt. I've been testing it locally with a lower time but it would make sense for the code to be flexed regularly in the system-tests.